src/styles/main.css

```diff
@@ -1,3 +1,7 @@
+body{
+  background-color: #f6f4f4eb;
```

Comment:
can we use a token name for the color being used here?

Reply:
Created CSS variable 🥇